src/resources/views/fields/table.blade.php

```diff
@@ -140,13 +140,13 @@
                         }
                     }
 
-                    if( typeof $scope.items != 'undefined' && $scope.items.length ){
+                    if( typeof $scope.items != 'undefined' ){
```

Comment:
https://docs.angularjs.org/api/ng/function/angular.isUndefined

(but I suppose this works too)